tests/templates/test_subroutines/test_qubitization.py

```diff
@@ -196,12 +197,13 @@ def test_qnode_torch(self, shots, seed):
 
         dev = qml.device("default.qubit", shots=shots, seed=seed)
         diff_method = "backprop" if shots is None else "parameter-shift"
+        atol = 1e-5 if shots is None else 0.05
         qnode = qml.QNode(self.circuit, dev, interface="torch", diff_method=diff_method)
 
         params = torch.tensor(self.params, requires_grad=True)
         jac = torch.autograd.functional.jacobian(qnode, params)
         assert qml.math.shape(jac) == (4,)
-        assert qml.math.allclose(jac, self.exp_grad, atol=0.01)
```

Comment:
@mudit2812 @andrijapau According to our discussion, the flaky here inside `test_qubitization.py` is also something with unreasonable tol. I also noticed that in the same test class, we first tested using autograd with `tol=1e-5` on analytic mode but much lowered for Jax or Torch. So I also refined the tol there to be `1e-5` as well. For finite-shot mode I set `0.05` for both.
Didn't touch TF since it's xfailed 1year ago looks scary.